package.json

```diff
@@ -21,5 +21,6 @@
     "request": "^2.76.0",
     "request-promise": "^4.1.1",
     "socket.io-client": "^1.5.1"
+  	"ansi-color": "latest"
```

Comment:
Usually, I don't think this is a good idea because if the latest version has breaking changes things go bad. But this is a colour package so I don't expect any big changes to come